spec/features/authentication_pages_spec.rb

```diff
@@ -1,41 +1,41 @@
 require "rails_helper"
 
-describe "Authentication" do
+feature "Authentication" do
 
   subject { page }
 
-  describe "signin page" do
+  feature "signin page" do
     before { visit login_path }
 
-    it { should have_content('Log in') }
-    it { should have_title('Log in') }
+    scenario { should have_content('Log in') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.